pkgs/games/pegasus-frontend/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, cmake
+, libsForQt5
+, wrapQtAppsHook
+, SDL2
+, sqlite
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pegasus-frontend";
+  version = "unstable-2023-05-22";
+
+  src = fetchFromGitHub {
+    owner = "mmatyas";
+    repo = "pegasus-frontend";
+    rev = "6421d7a75d29a82ea06008e4a08ec14e074430d9";
+    sha256 = "sha256-mwJm+3zMP4alcis7OFQUcH3eXlRTZhoZYtxKrvCQGc8=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake wrapQtAppsHook ];
```

Comment:
Also might want to have one dependency per line for consistency and cleaner diffs in future.

Reply:
alright done, will also restructure the commit messages later. So I should be OK with these two commits then (?):

1. "maintainers: add tengkuizdihar" <- changes on `maintainer-list.nix`
2. "pegasus-frontend: init at unstable-2023-05-22" <- changes on everything else

Reply:
Yeah, that sounds good. Though I accidentally linked to an old version of the manual. According to the [latest one](https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming), now the version would be `0-unstable-2023-05-22` to denote there is no stable version.

Reply:
done restructuring the commit and changing the version @jtojnar